release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4234,3 +4234,35 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-13-44-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-45"]
+=== RHSA-2024:4484 - {product-title} 4.13.45 bug fix and security updates
+
+Issued: 2024-07-17
+
+{product-title} release 4.13.45, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4484[RHSA-2024:4484] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4486[RHSA-2024:4486] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.45 --pullspecs
+----
+
+[id="ocp-4-13-45-bug-fixes"]
+==== Bug fixes
+
+* Previously, a bug in `growpart` caused the device to lock which, prevented the LUKS-encrypted devices from opening. The system would fail to boot successfully. With this release, `growpart` was removed from the process and the the system will boot successfully. (link:https://issues.redhat.com/browse/OCPBUGS-35990[*OCPBUGS-35990*])
+////
+* Previously, the {azure-first} node controller container did not tolerate the `NoExecute` taint on nodes. This caused a condition where a node would be uninitialized. With this release, the node controller deployment receives an update to tolerate the `NoExecutre` taint, so that nodes can be properly initialized. (link:https://issues.redhat.com/browse/OCPBUGS-35562[*OCPBUGS-35562*])
+////
+
+* Previously, the `wait-for-ceo` command used during the bootstrapping process to verify etcd rollout would not report errors for some failure modes. With this release, those error messages should now be visible on the `bootkube` script if the command exits in an error case. (link:https://issues.redhat.com/browse/OCPBUGS-35077[*OCPBUGS-35077*])
+
+* Previously, the Machine API controller could not determine the zone of machines in vSphere clusters that use multiple zones. With this release, the zone lookup logic is based on the host of a VM and, as a result, machine objects indicate proper zones. (link:https://issues.redhat.com/browse/OCPBUGS-32015[*OCPBUGS-32015*])
+
+[id="ocp-4-13-45-updating"]
+==== Updating
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Comment:
```suggestion
To update an {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
In many cases, you can remove the word _existing_ and retain your intended meaning.

Reply:
This is correct and another thing that's the same with every z-stream.